### PR TITLE
chore: use out.String() instead of string(out.Bytes())

### PR DESCRIPTION
**Pull request by testwill**
**What type of PR is this?**
> /kind cleanup

**What this PR does / why we need it**:
should use out.String() instead of string(out.Bytes())
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/ecr-credential-provider/plugin_test.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ func Test_runPlugin(t *testing.T) {
 			}
 
 			if !reflect.DeepEqual(out.Bytes(), testcase.expectedOut) {
-				t.Logf("actual output: %v", string(out.Bytes()))
+				t.Logf("actual output: %v", out.String())
```
Comment (cartermckinnon) [inline cmd/ecr-credential-provider/plugin_test.go:95]: Is this the only occurrence of this?

Comment (testwill) [reply cmd/ecr-credential-provider/plugin_test.go:95]: yes

**Review by cartermckinnon — APPROVED** [review]

**Comment by cartermckinnon** [comment]
This is a [trivial edit](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits), but I don't see any additional cleanup we can do to bulk it up.

/ok-to-test
/kind cleanup
/triage accepted

**Review by hakman — APPROVED** [review]

**Pull request merged**